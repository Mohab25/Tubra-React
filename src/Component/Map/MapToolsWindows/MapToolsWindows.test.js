import React from 'react'
import {render,screen} from '@testing-library/react'
import MapToolsWindow from './MapToolsWindows'

describe('rendering test',()=>{
    it('renders correctly - snapshot test',()=>{
        expect(render(<MapToolsWindow display='flex'/>)).toMatchSnapshot()
    })
})