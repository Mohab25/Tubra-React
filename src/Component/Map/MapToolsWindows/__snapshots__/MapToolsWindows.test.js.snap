// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering test renders correctly - snapshot test 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MapToolsWindow"
        style="display: flex;"
      >
        <div
          class="MapToolsWindowContainer"
        >
          <div
            class="MapToolsWindowTitleContainer"
          >
            <h6>
              Tools
            </h6>
            <i
              class="fa fa-times cancel-icon"
            />
          </div>
          <div
            class="ToolsWindowContainer"
          >
            <div
              class="Tiles-control-window"
            >
              <div
                class="Tiles-control-window-container"
              >
                <div
                  class="preview-container"
                >
                  <div
                    class="preview-image"
                    style="background-image: url(Dark.jpg);"
                  />
                  <p
                    class="preview-text"
                  />
                  <div
                    class="preview-image"
                    style="background-image: url(Light.jpg);"
                  />
                  <p
                    class="preview-text"
                  />
                  <div
                    class="preview-image"
                    style="background-image: url(Topo.jpg);"
                  />
                  <p
                    class="preview-text"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MapToolsWindow"
      style="display: flex;"
    >
      <div
        class="MapToolsWindowContainer"
      >
        <div
          class="MapToolsWindowTitleContainer"
        >
          <h6>
            Tools
          </h6>
          <i
            class="fa fa-times cancel-icon"
          />
        </div>
        <div
          class="ToolsWindowContainer"
        >
          <div
            class="Tiles-control-window"
          >
            <div
              class="Tiles-control-window-container"
            >
              <div
                class="preview-container"
              >
                <div
                  class="preview-image"
                  style="background-image: url(Dark.jpg);"
                />
                <p
                  class="preview-text"
                />
                <div
                  class="preview-image"
                  style="background-image: url(Light.jpg);"
                />
                <p
                  class="preview-text"
                />
                <div
                  class="preview-image"
                  style="background-image: url(Topo.jpg);"
                />
                <p
                  class="preview-text"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
